src/read_write/s2.rs

```diff
@@ -160,11 +160,12 @@ where
         }
     }
 
-    pub fn get_meta(self) -> S2Meta {
-        S2Meta::new(
+    pub fn get_meta(self) -> Option<S2Meta> {
```

Comment:
Again, a sort of orthogonal change.

Reply:
I don't understand this change. It just always returns `Some` and never `None`?

Reply:
There's a `?` after bounding_box.